src/main/java/org/activiti/cloud/qa/story/TweeterConnectorActions.java

```diff
@@ -42,7 +44,51 @@ public void matchingTweet() {
         tweeterSteps.tweet(lastSentMatchingTweet);
     }
 
+    @When("a non-matching tweet is sent")
+    public void nonMatchingTweet(){
+        lastSentNonMatchingTweet = new Tweet("A day without sunshine is like, you know, night." + System.currentTimeMillis(),
+                                             "paul",
+                                             "en");
+        tweeterSteps.tweet(lastSentNonMatchingTweet);
+    }
+
+    @When("a positive tweet is sent")
+    public void tweetRankedAsPositive(){
+        lastRankedTweet = new Tweet("A goal is a dream with a deadline... Also Activiti Cloud is great! " + System.currentTimeMillis(),
+                                        "paul_good_mood",
+                                        "en");
+        tweeterSteps.tweet(lastRankedTweet);
+
+    }
+
+    @When("a neutral tweet is sent")
+    public void tweetRankedAsNeutral(){
+        lastRankedTweet = new Tweet("Does anyone know the guys behind Activiti?" + System.currentTimeMillis(),
+                                        "paul_neutral_mood",
+                                        "en");
+        tweeterSteps.tweet(lastRankedTweet);
+
+    }
+
+    @When("a negative tweet is sent")
+    public void tweetRankedAsNegative(){
+        lastRankedTweet = new Tweet("I hate Activiti forkers and everything related to BPM " + System.currentTimeMillis(),
```

Comment:
I'd suggest to use a more generic subject (bad weather or similar).

Reply:
The message needs to be related to Activiti. Otherwise, the tweet will end up discarded and never processed by the processing service, therefore never ranked as negative.
We can tune (or soften) the negative message, tough.
However, we need to make sure that it gets ranked as negative and falls within the now broader filter that the PR in the processing service was addressing.
Open to suggestions ;)